src/scripts/apiScripts.js

```diff
@@ -0,0 +1,50 @@
+import cookie from 'react-cookies'
+
+export const BACK_AVAILABLE = ( process.env.REACT_APP_BACK_API_AVAILABLE == 'true' ? true : false ); 
```

Comment:
Talked about this in our standup, do this need to be exported?
```suggestion 
const BACK_AVAILABLE = ( process.env.REACT_APP_BACK_API_AVAILABLE == 'true' ? true : false ); 
```